src/utils/UndoableActionsHandlerProvider.js

```diff
@@ -92,10 +92,19 @@ const UndoableActionsHandlerProvider = ({ children }) => {
         if (Object.keys(actions).length && !closeLock) {
             setCloseLock(true);
             initialBeforeUnloadEventListener.current = window.onbeforeunload;
-            window.onbeforeunload =  () => "Some changes might have not taken effect yet...";
+
+            Object.defineProperty(window, "onbeforeunload", {
+                value: () => "Some changes might have not taken effect yet...",
+                writable: true,
```

Comment:
why the change?

Reply:
I believe there have been some changes in some of our dependencies that made this test (and other similar ones that are commented as well so I'm gonna mark them as resolved) fail with a runtime error, saying that "onbeforeunload" is not assignable. This fixes that.